tests/test_firefox_urls.py

```diff
@@ -37,6 +37,11 @@ class TestFirefoxURLs(object):
         '/firefox/tiles/',
         '/firefox/unsupported-systems/',
         '/firefox/unsupported/EOL/',
+        # Legacy URLs
+        '/firefox/start/central.html',
+        '/firefox/en-US/firefox/whatsnew/',
```

Comment:
I don't think we need this any more; this redirect was removed as part of https://github.com/mozilla/bedrock/pull/2643 which fixed the root cause for traffic to that bad url in the first place.